### PR TITLE
Add Authentication Service parameter to Heroku Deploy

### PR DESCRIPTION
**Pull request by HazyArc14**
This pull request includes a

- [ ] Bug fix
- [x] New feature
- [ ] Translation

The following changes were made

- Added ability for user to explicitly specify an Authentication Service in Heroku Deploy

### CODE REVIEW

**Comment by HazyArc14** [comment]
The wiki page for Heroku Deployment will need to be updated to:

    # Get a local copy of the project
    git clone https://github.com/AHAAAAAAA/PokemonGo-Map.git

    # Create a Heroku app for the project
    heroku apps:create

    # Set the required configuration parameters for the app, replace with your
    # actual values.
    heroku config:set AUTH_SERVICE=insertyourauthservicehere
    heroku config:set USERNAME=insertyourusernamehere
    heroku config:set PASSWORD=insertyourpasswordhere
    heroku config:set LOCATION=insertyourlocationhere
    heroku config:set STEP_COUNT=insertyourstepcounthere
    heroku config:set GMAPS_KEY=inseryourgmapskeyhere

    # Push the project develop branch to the master branch on Heroku
    git push heroku develop:master

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: python runserver.py -u "$USERNAME" -p "$PASSWORD" -l "$LOCATION" -st $STEP_COUNT -H 0.0.0.0 -P $PORT -k $GMAPS_KEY $EXTRA_ARGS 
+web: python runserver.py -a "AUTH_SERVICE" -u "$USERNAME" -p "$PASSWORD" -l "$LOCATION" -st $STEP_COUNT -H 0.0.0.0 -P $PORT -k $GMAPS_KEY $EXTRA_ARGS
```
Comment (billyvg) [inline Procfile]: `AUTH_SERVICE` is a string -- also you could just use `EXTRA_ARGS`

**Comment by HazyArc14** [comment]
I fixed the problem with the `AUTH_SERVICE`.
Yes, you can just use `EXTRA_ARGS` but I figured that this would be better since you would be using one or the other.

**Comment by MonkeyLeeT** [comment]
Looks good. Merged.

**Pull request merged**